pyspectral/rayleigh.py

```diff
@@ -301,6 +280,20 @@ def reduce_rayleigh_highzenith(zenith, rayref, thresh_zen, maxzen, strength):
         return rayref * factor
 
 
+def check_and_normalize_sensor(platform_name: str, sensor: str) -> str:
```

Comment:
I think this function could be called simply `normalize_sensor`, I consider checking as a natural part of normalisation.